magento/pwa-studio
# Customer Account - Newsletter Subscription

## Description

Create Newsletter Subscription page for Customer account.

## Related Issue





Closes #2570.

## Acceptance 




### Verification Stakeholders

### Specification



### Verification Steps


1. SignIn an account you registered.
2. After login success, you will found title: Newsletter Subscription. Click Newsletter Subscription to see Newsletter Subscription page.
3. Here, you can manage your newsletter subscription here.

## Screenshots / Screen Captures (if appropriate)
_Newsletter subscription menu:_

![newslettermenu](https://user-images.githubusercontent.com/9253798/88159495-913a3500-cc37-11ea-8694-f1f22bbc37c6.png)

_Newsletter subscription management page:_
![newsletter_subscription](https://user-images.githubusercontent.com/9253798/88159715-e37b5600-cc37-11ea-9852-11906a144680.png)

## Checklist

* I have updated the documentation accordingly, if necessary.
